### DISCUSSION TITLE
Issue mentions' background color based on whether they are open / closed.

### DISCUSSION THREAD

**Discussion by zoomlogo** [community/community#5527] [Projects and Issues] [answered]
The issue background color can be red / green depending on whether its opened / closed. Pretty much like gitter.

So:
#<issue_number> will have a background color of green if the issue is open.

**Comment by ajsirota** [comment, marked as answer]
This extension might do what you want https://github.com/fregante/github-issue-link-status

**Reply by zoomlogo** [reply]
thank you!
